BattleNetwork/bindings/bnScriptEnvironmentManager.h

```diff
@@ -16,7 +16,11 @@ class ScriptEnvironmentManager
     static void ConfigureEnvironment( sol::state& state );
 
     private:
-    
+    static void DefineObject_Tile( sol::state& state );
```

Comment:
underscores are not a good function naming scheme for the project

Reply:
I'll keep that in mind for the future.
This file doesn't exist anymore, though. Rolled back into `bnScriptResourceManager.cpp` to avoid unnecessary merge conflicts.